src/datamodel/quest/quest.rs

```diff
@@ -1,9 +1,8 @@
-use crate::datamodel::quest::quest_member::QuestMember;
 use crate::datamodel::quest::quest_objective::QuestObjective;
-use crate::datamodel::quest::quest_proposer::QuestProposer;
+use crate::datamodel::Player;
 
-pub struct Quest {
-    objective: QuestObjective,
-    proposer: QuestProposer,
-    members: Vec<QuestMember>,
+pub struct Quest<'a> {
+    pub objective: QuestObjective,
+    pub proposer: &'a Player,
```

Comment:
I wonder if `Player` shouldn't be a `mut&` instead of just an immutable reference. what do you think?

Reply:
I don't think we intend to mutate this. One of the aims of the design was to keep mutability at a minimum, and concentrated in the manager classes. All these simple classes are supposed to be more like data containers: small, simple, without behaviour, and immutable. This will make them more suitable for re use, testing, and adhere to SOLID principles.

Reply:
Agreed. I didn't know about Rc when I wrote the comment. My concern here was on ownership.

Reply:
Same as the other comment: are you OK with merging as it is and revisiting the use of references and smart pointers later?

Reply:
Yep